.github/workflows/testing-workflow.yml

```diff
@@ -41,4 +41,4 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Check changes in Debezium Testing
-        run: mvn clean install -B -pl debezium-testing -am -amd -Passembly -Dcheckstyle.skip=true -Dformat.skip=true -Drevapi.skip -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: mvn clean install -B -pl debezium-assembly-descriptors,debezium-core,debezium-testing -am -amd -Passembly -Dcheckstyle.skip=true -Dformat.skip=true -Drevapi.skip -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
I needed to fix the Testing Build as dependencies were not build properly. 
see also https://github.com/debezium/debezium/pull/1920/files#r539168346

Reply:
Does this address the separate DBZ issue you had logged for this?

Reply:
This is not that easy. As you introduced core and assembly descriptors dependency and there is `-amd` option the net result is that the full Debezium is build (incl. tests) not only those we are inetrested in.

So IMHO the correct approach for this case would be execution of Maven twice. First run will build the core and assembly descriptors and the second will be the same as is now and wil use those built artifacts.

Unfortunately if we omit `-amd` then we would need to enumerate the server modules in project list.

Reply:
That's why I filed https://issues.redhat.com/browse/DBZ-2861 to properly solve that. Currently we have 2 issues: Either the build is failing because an artifact is missing in the Maven cache repo and the job doesn't build that dependency (like here with the missing `debezium-assembly-descriptors` jar. Or the build has a cached jar, but in-fact should use new updated code instead of the old cached one.

Reply:
I will apply this PR without this change, as it's discussed separately via DBZ-2861.